src/methods/using_methods/hard/q5/Standing.java

```diff
@@ -0,0 +1,39 @@
+package methods.using_methods.hard.q5;
+
+/*
+In University your standing is designated by which numeric year you are in. Write a Java Program called Standing that
```

Comment:
Change text to:
`
In university, your standing is designated by which numeric year you are in. Write a Java Program called Standing that asks users what year they are in university. Then, write a method called designationCalc that accepts that integer as an input parameter and returns a string designating the user's university standing. If the user is in their first year, they're freshmen; if the user is in their second year, they're a sophomore; if the user is in their third year, they're a junior; if the user is in their fourth year or above, they're a senior. If the input is anything else, the year is unknown.
`